### PR TITLE
chore: CE-556 HWCR drug administration information updates

### PR DESCRIPTION
**Pull request by joshgamache**
Addresses [CE-556](https://env-ceds.atlassian.net/browse/CE-556). Changes to some fields and available options when entering Drug information for animals. 

**Note**: This PR is paired with an [equivalent PR in Case Management](https://github.com/bcgov/nr-compliance-enforcement-cm/pull/101).

## Changes 🚧

- Added _Oral/nasal_ to options in 'Injection method'.
- Replaced an option in 'Fate of remaining...' with _Returned to vet_. 
- Removed _Adverse reactions_, _Discard method_, and _Amount discarded_ fields.
- Added _Additional comments_ text field.
- (In CM DB) Migrated removed fields' data into _Additional comments_ data. 

## How Has This Been Tested? 🔬

- [x] Manual testing, matching  from issue report.
- [x] Dummy data in DB for migration.

### To test 🔬

0. Ensure both this and `...cm` repo are on this issue's branch.
1. Run a local instance of the application, log in. 
2. Navigate to a HWC complaint and **Add animal**, then **Add drug**.
   - You should no longer see the removed fields. You should now see an 'Additional comments' open text field. 
   - Under 'Injection Method', you should now see a fourth option _Oral/nasal_
   - Under 'Fate of remaining drug in vial', you should not see _Reverse distribution_ but instead see _Returned to vet_
3. Fill out the fields, and save the 'Animal outcome'. 

### Checklist 📃

- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

[CE-556]: https://env-ceds.atlassian.net/browse/CE-556?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

---

Thanks for the PR!

Deployments, as required, will be available below:
- [Frontend](https://nr-compliance-enforcement-740-frontend.apps.silver.devops.gov.bc.ca)
- [Backend](https://nr-compliance-enforcement-740-frontend.apps.silver.devops.gov.bc.ca/api)

Please create PRs in draft mode.  Mark as ready to enable:
- [Analysis Workflow](https://github.com/bcgov/nr-compliance-enforcement/actions/workflows/analysis.yml)

After merge, new images are deployed in:
- [Merge Workflow](https://github.com/bcgov/nr-compliance-enforcement/actions/workflows/merge.yml)

### CODE REVIEW

File: backend/src/types/models/case-files/wildlife/drug-used-input.ts [hunk 1]
```diff
@@ -11,3 +11,15 @@ export interface DrugUsedInput {
...
+  id?: number;
+
+  vial: string;
+  drug: string;
+  amountUsed: string;
+  injectionMethod: string;
+
+  remainingUse: string;
+  additionalComments: string;
+}
```
Comment (afwilcox) [inline backend/src/types/models/case-files/wildlife/drug-used-input.ts:25]: Isn't this creating dead code as no one else is using DrugUsedInput?

File: frontend/src/app/types/app/complaints/outcomes/wildlife/drug-used.ts [hunk 2]
```diff
@@ -35,3 +35,17 @@ export interface DrugUsedV2 {
 
   order: number;
 }
+
+export interface DrugUsedV3 {
```
Comment (afwilcox) [inline frontend/src/app/types/app/complaints/outcomes/wildlife/drug-used.ts:39]: Same comment as above - probably don't need multiple versions here if the previous versions are dead code

**Review by afwilcox — CHANGES_REQUESTED** [review]
I know that this wasn't part of this, but we can probably reduce the duplication a bit by exporting some of the duplicated code.

File: frontend/src/app/types/app/complaints/outcomes/wildlife/drug-used.ts (hunk 2 above)
Comment (joshgamache) [reply frontend/src/app/types/app/complaints/outcomes/wildlife/drug-used.ts]: Cleaned up the types with b5d83d8.

File: backend/src/types/models/case-files/wildlife/drug-used-input.ts (hunk 1 above)
Comment (joshgamache) [reply backend/src/types/models/case-files/wildlife/drug-used-input.ts]: Cleaned up the types with b5d83d8.

**Review by afwilcox — APPROVED** [review]
Going to move this one along since the remaining issues are all a result of legacy code

**Pull request merged**